setup.py

```diff
@@ -161,11 +161,16 @@ def install_requires():
 
 class JaxBuilder(FrameworkBuilderBase):
     def cmake_flags(self):
-        return ["-DENABLE_JAX=ON"]
+        p = [d for d in sys.path if 'dist-packages' in d][0]
+        return ["-DENABLE_JAX=ON", "-DCMAKE_PREFIX_PATH="+p]
 
     def run(self, extensions):
         print("Building jax extensions!")
 
+    def install_requires():
+        # TODO: need to find a way to instal directly pybind11 and ninja
+        return ['pydantic', 'cmake']
```

Comment:
Do we need cmake from pip?

Reply:
Also, do we actually need pydantic? Its functionality does not seem necessary for supporting TE in JAX?

Reply:
cmake could come from the system. But having it installed from python simplify end user experience.

I think pydantic is needed due to this code:
```
$ git grep pydantic
setup.py:        return ['pydantic', 'cmake']
transformer_engine/common/recipe.py:from pydantic.dataclasses import dataclass
```

Reply:
@ksivaman Since pydantic is used in common, that should actually be a dependency for all frameworks, right?

Reply:
If we're getting CMake from pip, I wonder if that gives us a nicer way to get the CMake executable.